### PR TITLE
Fix missing files in fxmanifest

### PR DESCRIPTION
**Pull request by ADRNALN**
## Description

Fixes issue below
image

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: fxmanifest.lua [hunk 1]
```diff
@@ -26,7 +26,9 @@ server_scripts {
 
 files {
     'config/shared.lua',
+    'config/client.lua',
+    'config/server.lua'
```
Comment (dusan01) [inline fxmanifest.lua]: Remove config/server.lua as is not needed.

Comment (ADRNALN) [reply fxmanifest.lua]: Why would it not be needed? server/main.lua calls it

Comment (solareon) [reply fxmanifest.lua]: The server always has access to all the files within a resources directory. The files directive only tells the client what they can see and download

Comment (mafewtm) [reply fxmanifest.lua]: This was my mistake in my latest commit so my apologies but to answer your question, `files {}` are files downloaded by the client upon resource start. server files are not needed in here. server/main.lua will see it upon resource start.

**Review by Manason — APPROVED** [review]

**Review by mafewtm — APPROVED** [review]

**Pull request merged**